x.py

```diff
@@ -257,8 +257,8 @@ def test_go(dir: str, rest: List[str]) -> None:
 
     parser_check = subparsers.add_parser(
         'check',
-        description="Check code with cpplint or cppcheck",
-        help="Check code with cpplint or cppcheck")
+        description="Check code with cpplint or cppcheck or golangci-lint",
+        help="Check code with cpplint or cppcheck or golangci-lint")
```

Comment:
```suggestion
        description="Check or lint source code",
        help="Check or lint source code")
```

I think it's time to stop enumerating candidates, they should already show up in the help message:

```
./x.py check
usage: x.py check [-h] {cpplint,cppcheck} ...

Check code with cpplint or cppcheck

positional arguments:
  {cpplint,cppcheck}
    cpplint           Lint code with cpplint
                      (https://github.com/cpplint/cpplint)
    cppcheck          Check code with cppcheck
                      (https://github.com/danmar/cppcheck)

options:
  -h, --help          show this help message and exit
```

Reply:
Can't agree more.